code/index.html

```diff
@@ -1,30 +1,93 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>🐶 Dogpool</title>
+</head>
+<body>
+    <header> 
+      
+        <div class="header-video" style="width: 1349px; height: 800px;">
+             <video autoplay="true" loop="true" muted="" id="header-video"> type="video/mp4">
+            <source src="C:\Users\siri_\Desktop\KodÖvEx\Pexels Videos 2776522.mp4" type="video/mp4"></video></div>
+         <div class="overload"> <h1>The Dogpool🐕 </h1> <h2>For healthy, happy puppys!</h2>
+        </div> 
+        </div>
+     </header>
+    
+    <section class="container"> 
+        <section class="sign-up-form">
+           <h4>Sign up for our classes here! 🥇</h4>
+           <form action="http://httpbin.org/anything" method="post">
+           <label>
+             <input type="text" name="yourName" placeholder="Your Name" required>
+           <label>
+     
+           <label>
+             <input type="text" name="dogsName" placeholder="Dogs Name" required>
+           <label>
+           
+           <label>
+             <input type="email" placeholder="name@email" required>
+           <label>
+           
+           <label>
+           <select name="Swimclasses">
+               <option value="Rehab30">Rehab 30 minutes</option>
+               <option value="Rehab60">Rehab 60 minutes</option>
+               <option value="Exercise30">Exercise 30 minutes</option>
+               <option value="Exercise60">Exercise 60 minutes</option>
+             </select>
+             <label>
 
-    <!-- Signup form -->
+             <label>
+               <input type="radio" name="Dogsize" value="BigDog" > Big dog
+                <input type="radio" name="Dogsize" value="MediumDog"> Medium dog
+                <input type="radio" name="Dogsize" value="SmallDog"> Small dog 
+             </label>
+     
+             <label>
+                <button class="bone">
+                    <div class="c1"></div>
+                    <div class="c2"></div>
+                    <div class="c3"></div>
+                    <div class="c4"></div>
+                    <div class="b1">
+                      <div class="b2">
+                        Submit
+                      </div>
```

Comment:
Try to name your classes so its more as a description of the element. It's hard to know what `c3` stands for!